README.md

```diff
@@ -81,7 +109,12 @@ body that says "hello".
 
 Your Answer:
 ```js
-// your code here
+
+$("#greeting").on("click", function(){
+  newParagraph = $("p").append("hello")
+  $("body").append(newParagraph)
+})
+
```

Comment:
hmm, did you run this in the console? I'm not sure it works.. 
what about something like:
```$("body").append("<p>hello</p>")```